frontend_tests/zjsunit/zpage_params.js

```diff
@@ -1,11 +1,17 @@
 "use strict";
 
 exports.page_params = {};
+exports.user_settings = {};
 
 exports.reset = () => {
     for (const field in exports.page_params) {
         if (Object.prototype.hasOwnProperty.call(exports.page_params, field)) {
             delete exports.page_params[field];
         }
     }
+    for (const field in exports.user_settings) {
+        if (Object.prototype.hasOwnProperty.call(exports.user_settings, field)) {
+            delete exports.user_settings[field];
+        }
+    }
```

Comment:
I added the `user_settings` code here as it works similarly to `page_params` in the node tests. Not sure this change is worth a separate file or we should change this file name.

Reply:
This looks great!